apps/epl_ets/src/epl_ets_metric.erl

```diff
@@ -0,0 +1,186 @@
+%%%-------------------------------------------------------------------
+%% @doc epl_ets_metric module.
+%% This module provides an APIs for gathering and processing ETS related
+%% metrics. It means information, data or statistics.
+%% @end
+%%%-------------------------------------------------------------------
+
+-module(epl_ets_metric).
+
+%% API
+-export([get_node_ets_num/1,
+         get_node_ets_mem/1,
+         get_node_ets_tabs/1,
+         get_ets_tabs_info/2,
+         get_ets_call_stats/1]).
+
+%% Test
+-export([split_traces_by/2,
+         sort_traces_by_ms/1,
+         clean_traces_unpaired/1,
+         calculate_call_time/1]).
+
+%%====================================================================
+%% API functions
+%%====================================================================
+
+%% @doc Calculates number of the ETS tables present on the `Node'.
+-spec get_node_ets_num(Node :: atom()) -> integer().
+get_node_ets_num(Node) ->
+    {ok, AllETS} = epl:command(Node, fun ets:all/0, []),
+    erlang:length(AllETS).
+
+%% @doc Calculates allocated memory usage for the ETS tables present on the 
+%% `Node'.
+-spec get_node_ets_mem(Node :: atom()) -> float().
+get_node_ets_mem(Node) ->
+    {ok, MemoryData} = epl:command(Node, fun erlang:memory/0, []),
+    MemoryPercent = proplists:get_value(ets, MemoryData) / 
```

Comment:
It's better to put such a computation to a separate function which name exactly explains what the function means.